migrations/2018_07_21_000100_seed_default_group_permissions.php

```diff
@@ -27,6 +27,7 @@
     ['permission' => 'discussion.hidePosts', 'group_id' => Group::MODERATOR_ID],
     ['permission' => 'discussion.rename', 'group_id' => Group::MODERATOR_ID],
     ['permission' => 'discussion.viewIpsPosts', 'group_id' => Group::MODERATOR_ID],
+    ['permission' => 'user.viewLastSeenAt', 'group_id' => Group::MODERATOR_ID],
```

Comment:
Adding this to the existing migration means we need to add this as a manual action to the upgrade guide, right? Or am I missing something?

Reply:
@franzliedke Not sure... I guess that'd be something to test along with everything else for a beta 7 -> 8 conversion because of merging db changes...

Reply:
@franzliedke As it's a new file, I assume there'd be no problems... Flarum will load the whole thing on new or updated installation.

Reply:
Yeah, this migration is dated after beta 7 release.